pgml-dashboard/src/main.rs

```diff
@@ -29,3 +29,5 @@ async fn main() {
         .await
         .expect("failed to shut down Rocket");
 }
+
+mod tests;
```

Comment:
You can put the tests in the same file as the code being tested. This is the Rust way, tests and code together. Rust also has integration tests, which go into a separate app that then uses public methods of the crate it's testing.

Reply:
Moved tests to main.rs